src/shared/components/ProfilePage/Stats/HistoryGraph/index.jsx

```diff
@@ -58,6 +59,23 @@ export default class HistoryGraph extends React.Component {
     return 300;
   }
 
+  onHandleDataPointClicked() {
+    const { challengeId, href } = this.state;
+    fetch(`${config.API.V5}/challenges?legacyId=${challengeId}`)
```

Comment:
Not need call this API to get UUID from challenge, if access challenge with `legacyId` CA will redirect to UUID url:
example: 
- If try to access https://www.topcoder.com/challenges/16958, CA will redirect to https://www.topcoder.com/challenges/24bcb947-02d9-4e95-a4ff-bf820aaf1164

Reply:
Hmmm, never notice that. will change that.